docs/content/docs/dev/table/sqlClient.md

```diff
@@ -619,6 +619,65 @@ After execution, SQL Client will print the cluster id on the terminal. The scrip
 <span class="label label-danger">Attention</span> When deploying a script to the cluster, SQL Client only supports running with `--jars` startup option, other options, e.g. `--init` 
 are not supported.
 
+### Execute Materialized Table Refresh Jobs in Kubernetes/YARN
+
+#### Execute Refresh Jobs in Session Mode
```

Comment:
nit: I suggest describing why a `CREATE MATERIALIZED TABLE` produces a refresh job, or pointing to the details here. Is there a consideration around whether the table exists or not.

Reply:
This section has already been introduced on the overview page: https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/dev/table/materialized-table/overview/